src/App.tsx

```diff
@@ -1,6 +1,6 @@
-import React from 'react';
-import Dashboard from './components/dashboard';
-import Login from './components/login';
+import "./App.css";
```

Comment:
**nitpick**: Consider using absolute imports for better readability and maintainability, especially as the project grows.